pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -2175,12 +2177,10 @@ func buildListener(opts buildListenerOpts) *listener.Listener {
 	return listener
 }
 
-// appendListenerFallthroughRoute adds a filter that will match all traffic and direct to the
-// PassthroughCluster. This should be appended as the final filter or it will mask the others.
-// This allows external https traffic, even when port the port (usually 443) is in use by another service.
-func appendListenerFallthroughRoute(l *listener.Listener, opts *buildListenerOpts,
-	node *model.Proxy, currentListenerEntry *outboundListenerEntry) {
-	wildcardMatch := &listener.FilterChainMatch{}
+// Create pass through filters for the listener assuming all the other filter chains are ready.
```

Comment:
would it be more accurate to say pass through filter chains, rather than filters?

Reply:
Good catch! done